js/packages/glimmer-idc/src/components/list-item.ts

```diff
@@ -0,0 +1,72 @@
+import Component, { hbs, tracked } from '@glimmerx/component';
+import MissingAsset from './missing-asset';
+
+interface Args {
+  listItem: ListItemObject,
+}
+
+interface ListItemObject {
+  uuid: string,
+  nid: number,
+  title: string,
+  field_description: string,
+}
+
+interface JsonApiResponse {
+  data: {}[],
+  included: {
+    attributes: {
+      uri: {
+        url: string
+      }
+    }
+  }[]
+}
+
+export default class ListItem extends Component<Args> {
+  @tracked imageUrl: string = null;
+
+  constructor(owner: unknown, args: Args) {
+    super(...arguments);
+
+    this.fetchThumbnail();
+  }
+
+  async fetchThumbnail() {
+    let url = `https://islandora-idc.traefik.me/jsonapi/media/image?include=thumbnail&filter[media-filter][condition][value]=${this.args.listItem.uuid}&filter[media-filter][condition][operator]=IN&filter[media-filter][condition][path]=field_media_of.id&filter[thumb-filter][condition][value]=Thumbnail Image&filter[thumb-filter][condition][operator]=IN&filter[thumb-filter][condition][path]=field_media_use.name`;
```

Comment:
Same comment as above, eventually we should make the URL relative, so changes to domain don't break things (i.e. deploying to different environments)